.github/workflows/daily-dependency-check.yml

```diff
@@ -0,0 +1,30 @@
+name: Daily dependency check
```

Comment:
This seems reasonable - for a second I was wondering why it hadn't fired (I arrived slightly too early because BST?) and then remembered that I believe schedules only trigger in the primary branch, so this should start running once merged 🤞

Reply:
🤞